docs/framework/migration-guide/reference-assemblies.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Build apps against Microsoft.NETFramework.ReferenceAssemblies"
+description: "Developers can target .NET Framework by referencing the official reference assemblies NuGet package."
+ms.date: 11/25/2022
```

Comment:
I altered the description a bit. I think this is more search engine friendly. I also updated the article date 😁 
```suggestion
description: "Learn how to reference the .NET Framework Reference Assemblies in your project, when you can't install a matching developer pack."
ms.date: 01/19/2023
```

Reply:
👍